.github/workflows/integrate.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: python -m pip install --force setuptools wheel
 
       - name: Install pipenv / poetry
-        run: python -m pip install pipenv==2021.11.5 poetry
```

Comment:
Maybe at this point we could remove the pinned pipenv version here - I added it temporarily to stabilize tests and be able to merge other PRs